src/patternfly/components/Alert/alert.scss

```diff
@@ -119,22 +125,54 @@
     --pf-c-alert--BoxShadow: var(--pf-c-alert--m-inline--BoxShadow);
     --pf-c-alert--BackgroundColor: var(--pf-c-alert--m-inline--BackgroundColor);
   }
+
+  &.pf-m-expandable {
+    grid-template-columns: var(--pf-c-alert--expandable--GridTemplateColumns);
+    grid-template-areas:
+      "toggleicon icon title action"
+      ". . description description"
+      ". . actiongroup actiongroup";
+  }
+
+  &.pf-m-expanded {
+    grid-template-columns: var(--pf-c-alert--expandable--GridTemplateColumns);
+    grid-template-areas:
+      "toggleicon icon title action"
+      ". . description description"
+      ". . actiongroup actiongroup"
+      ". . expandable-content expandable-content";
+
+    .pf-c-alert__toggle-icon {
+      grid-area: toggleicon;
+      background-color: var(--pf-c-alert--BackgColor);
+      transform: rotate(var(--pf-c-alert__icon--expanded__toggle-icon--Rotate));
+    }
+  }
+}
+
+.pf-c-alert__toggle-icon {
+  grid-area: toggleicon;
+  display: flex;
+  align-items: center;
 }
 
 .pf-c-alert__icon {
-  grid-area: icon;
   display: flex;
+  align-items: center;
   margin-top: var(--pf-c-alert__icon--MarginTop);
   margin-right: var(--pf-c-alert__icon--MarginRight);
   font-size: var(--pf-c-alert__icon--FontSize);
   color: var(--pf-c-alert__icon--Color);
+  grid-area: icon;
```

Comment:
nit - we use stylelint to lint our styles, including the order of CSS properties, though looks like the `grid-` properties' orders are not linted. I think it makes more sense to keep the `grid-` properties at the top near the `display` property, similar to how flex properties need to be grouped near the top. WDYT?

Reply:
If you agree, maybe we should move the `grid-template-areas/columns` applied to `.pf-c-alert` to the top, too.